api/src/main/java/org/pmiops/workbench/db/dao/WorkspaceDao.java

```diff
@@ -85,4 +86,44 @@ default void updateBillingStatus(long workspaceId, BillingStatus status) {
 
     Long getWorkspaceCount();
   }
+
+  @Query(
+      "SELECT\n"
+          + "  w.billingAccountType,\n"
+          + "  w.billingStatus,\n"
+          + "  w.cdrVersion.cdrVersionId AS cdrVersionId,\n"
+          + "  w.creationTime,\n"
+          + "  w.creator.userId AS creatorId,\n"
+          + "  w.disseminateResearchOther,\n"
+          + "  w.lastAccessedTime,\n"
+          + "  w.lastModifiedTime,\n"
+          + "  w.name,\n"
+          + "  w.needsResearchPurposeReviewPrompt AS needsRpReviewPrompt,\n"
+          + "  w.published,\n"
+          + "  w.additionalNotes AS rpAdditionalNotes,\n"
```

Comment:
based on your codegen, I suspect this SQL is generated. If so, how is one meant to update this to match changes in the codegen inputs? Also, should there be some kind of autogenerated comment that comes with it to indicate that this should not be modified manually?

Reply:
I should've called it a wizard and not a codegen tool, but you're right. It was on my list to do something like that, but I was worried I'd follow that rabbit too far and end up restructuring the whole thing.

Reply:
Standardizing column naming practices will help here, as well as best practices for entity classes.